composite-entity/src/test/java/com/iluwatar/compositeentity/PersistenceTest.java

```diff
@@ -0,0 +1,36 @@
+package com.iluwatar.compositeentity;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+/**
+ * Date: 4/17/21 - 11:45 AM
+ *
+ * @author zwebrain
+ */
```

Comment:
Please remove the author tag.

We can use git blame to know these metadata and avoid putting these to code

Reply:
I'm sorry. I have removed it.